index.js

```diff
@@ -17,6 +23,17 @@ function clean (db, done) {
           return coll.collectionName.indexOf('system') !== 0
         })
 
+        if (options !== null) {
+          if ('exclude' in options) {
+            // do not drop excluded collections
+            collections = collections.filter(function (coll) {
+              return (options.exclude.indexOf(coll.collectionName) === -1)
+            })
+
+            cb(null, db, collections)
+          }
+        }
```

Comment:
initialize an `exclude` variable at the top, initialized with an empty array, and always pass through the filter.